docs/docsite/rst/community/development_process.rst

```diff
@@ -143,7 +143,22 @@ We do not merge every PR. Here are some tips for making your PR useful, attracti
 Changelogs
 ----------
 
-Changelogs help users and developers keep up with changes to Ansible. Ansible builds a changelog for each release from fragments. You **must** add a changelog fragment to any PR that changes functionality or fixes a bug in ansible-core. You do not have to add a changelog fragment for PRs that add new modules and plugins, because our tooling does that for you automatically.
+Changelogs help users and developers keep up with changes to ansible-core and Ansible collections. Ansible and many collections build changelogs for each release from fragments. For ansible-core and collections using this model, you **must** add a changelog fragment to any PR that changes functionality or fixes a bug.
```

Comment:
@Andersson007 I don't think it's a good idea to remove any use of Ansible/ansible here, as it will destroy the meaning. Maybe ok would be:
```suggestion
Changelogs help users and developers keep up with changes to ansible-core and Ansible collections. Ansible and many collections build changelogs for each release from fragments. For them, you **must** add a changelog fragment to any PR that changes functionality or fixes a bug.
```
but even that doesn't sound right to me.

Reply:
we still have `.. Ansible and many collections..`. If we're talking about ansible-core it should be `ansible-core` but if we start with it, it should start with a capital letter:) Can we rephrase the sentence to put `ansible-core`, say, in the middle of the sentence?